### PR TITLE
Eb dev docs

### PR DESCRIPTION
**Pull request by emilymalecbrown**
Fleshing out docs! Would love edits around the adapter types and their types / return values.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -32,8 +31,46 @@ new AdDetective(config);
...
+adapters | No | Array | Array of the adapters listed in the table below | []
+mailTo | No | String | Email that will send information from reported ad | None 
+
+#### Adapter Types
+Adapter Name | Type | Return Values
+--- | --- | ---
+browser | String | Viewport, Url, Browser Version
+prebid | String | Ad Id, Bidder, Bid Amount, Ad Size
+dfp | String | Creative Id, Line Item ID
+tracking | Function | Key Value Pairs (correct me here!)
```
Comment (emilymalecbrown) [inline README.md]: not sure about this one

File: README.md [hunk 2]
```diff
@@ -1,21 +1,20 @@
...
+Please see [this example](samples/config.js) of a configuration object.
 
 ```javascript
 const config = {
-...TKTK
-};
+  selector: '.ad',
+  reasonInput: true,
+  reasonOptions: ['This ad offends me', 'This ad is great'],
+  mailto:'youremail@mediacompany.com',
```
Comment (trinhathu) [inline README.md]: Is it mailto or mailTo?

File: README.md [hunk 3]
```diff
@@ -1,21 +1,20 @@
...
-```
+A javascript library for information, identification, and reporting of ads. 
 
+Using this library on your site will enable users to report ads and provide detailed information about the ad and its context for debugging purposes. Right now this report will be send to the `mailto` link you provide in the config.
 
 ## Usage
+Please see [this example](samples/config.js) of a configuration object.
 
 ```javascript
 const config = {
```
Comment (trinhathu) [inline README.md]: As it stands right now, the config object is being set on the window as `window._adDetectiveConfig` without the need to call `new AdDetective(config);`. That might very well change with time, but let's update the docs to reflect that for now?

Comment (emilymalecbrown) [reply README.md]: Sounds like a plan!

File: README.md (hunk 2 above)
Comment (emilymalecbrown) [reply README.md]: mailto it is 😁

**Pull request merged**